src/adapter/pg.cr

```diff
@@ -77,4 +79,16 @@ class Granite::Adapter::Pg < Granite::Adapter::Base
       db.exec "DELETE FROM #{table_name} WHERE #{primary_name}=$1", value
     end
   end
+
+  private def _ensure_clause_template(clause)
```

Comment:
interesting, i like this!

Reply:
I do realize that this would make some functions of postgres inoperable, but we could also add an escape if you actually want a literal question mark in your query.

Doing a bit of digging last night I found that Laravel (php framework) does something similar and they just don't support question marks in postgres queries.

Just wanted to bring some light to this.

Reply:
I would hate to lose postgres functionality if we can avoid it.  However, I'm not sure we want to go down the escaping rabbit hole either.

Let's do this for now and if we see complaints regarding postgres and ?, we can evaluate a solution then.

Reply:
I'm having the same thoughts, sounds like this will be fine for now.